src/modm/math/geometry/vector2.hpp

```diff
@@ -57,10 +57,6 @@ namespace modm
 	{
 		friend class Location2D<T>;
 
-	public:
-		typedef typename GeometricTraits<T>::WideType WideType;
```

Comment:
This line is the issue. You removed the type alias which is used [here](https://github.com/modm-io/modm/blob/a33ee9d055b3ca1e53306bf67e4da9296bfebbf1/src/modm/math/geometry/vector2.hpp#L107). Thus, the compiler looks up the name `WideType` in an outer scope and finds the unrelated template `modm::WideType<T>`. It is trying to use it without template arguments and fails.